backend/app/models/prompt.py

```
@@ -0,0 +1,38 @@
+    description: str | None = Field(default=None, min_length=1, max_length=500)
+
+
+class Prompt(PromptBase, table=True):
+    id: int = Field(default=None, primary_key=True)
+    project_id: int = Field(foreign_key="project.id")
+    inserted_at: datetime = Field(default_factory=now, nullable=False)
+    updated_at: datetime = Field(default_factory=now, nullable=False)
+    is_deleted: bool = Field(default=False)
+    deleted_at: datetime | None = None
+
+    versions: list["PromptVersion"] = Relationship(back_populates="prompt")
```

Ruff (0.12.2)

Undefined name `PromptVersion`

(F821)